src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,151 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int currentCapacity = DEFAULT_CAPACITY;
+    private Node<K, V>[] values;
+    private int size = 0;
```

Comment:
You should avoid of " = 0".
Read about what value have class fields by default.